style.css

```
@@ -10,6 +10,18 @@
     --font-mono: 'JetBrains Mono', monospace;
 }
 
+.visually-hidden {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
```

Stylelint (17.11.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)